chart/compass/values.yaml

```diff
@@ -42,25 +42,25 @@ global:
       path: eu.gcr.io/kyma-project/incubator
     connector:
       dir:
-      version: "PR-1767"
+      version: "PR-1798"
```

Comment:
Since there are changes to the e2e tests, there should also be a PR number bump in the `e2e_tests` field below.